qiwis.py

```diff
@@ -140,6 +141,34 @@ class QiwiscallResult(Serializable):
     error: Optional[str] = None
 
 
+class MdiArea(QMdiArea):
+    """QMdiArea for the central widget.
+    
+    Attributes:
+        background: The QPixmap instance of the background image.
+    """
+
+    def __init__(self, background_path: Optional[str] = None):
+        """Extended.
+        
+        Args:
+            background_path: The path of the background image.
+        """
+        QMdiArea.__init__(self)

```

Comment:
Why don't you use `super().__init__()`?

Reply:
I couldn't understand it well, but it makes an error.
Also, the ChatGPT also told me the code above, so I did so.

Reply:
Could you explain in more detail? What errors did you get? What did ChatGPT say?

Reply:
Oh, I'm really sorry that I was mistaken.
`super().__init__()` works well and I will apply it.